trin-core/src/jsonrpc/utils.rs

```diff
@@ -36,6 +36,9 @@ pub fn bucket_entries_to_json(bucket_entries: BTreeMap<usize, Vec<NodeTuple>>) -
                         map.insert("enr".to_owned(), enr.to_base64());
                         map.insert("status".to_owned(), format!("{:?}", node_status.state));
                         map.insert("radius".to_owned(), format!("{}", data_radius));
+                        if let Some(client) = client_info {
+                            map.insert("client".to_owned(), client.to_string());
```

Comment:
If you were feeling fancy, you could expand the `t` to `trin` in the output

Reply:
Yea, I did that translation at first but removed it because I felt it would be brittle (across version and clients) for not much gain. But now I'm realizing I can just try to expand it, and if that fails for whatever reason just leave the original.